pages/osx/say.md

```diff
@@ -1,19 +1,23 @@
 # say
 
-> Uses text-to-speech to speak through the default sound device.
+> Converts text to speech.
 
 - Speak a phrase aloud:
 
-`say "I like to ride my bike."`
+`say {{"I like to ride my bike."}}`
 
 - Speak a file aloud:
 
-`say -f {{filename}}`
+`say -f {{filename.txt}}`
 
-- Create an AAC compressed audio file with the spoken text:
+- Speak a phrase with a custom voice and speech rate:
 
-`say -o {{filename.m4a}} "Everyone loves iTunes"`
+`say -v {{voice}} -r {{words_per_minute}} {{"I'm sorry Dave, I can't let you do that."}}`
 
 - List the available voices:
 
-`say -v '?'`
+`say -v ?`
+
+- Create an audio file of the spoken text:
+
+`say -o {{filename.aiff}} {{"Everyone loves iTunes."}}`
```

Comment:
Yeah I thought that one was weird -- that one is from the existing version of this page (that's definitely not something I'd write).